Makefile

```diff
@@ -4,27 +4,31 @@ PREFIX ?= $(DESTDIR)/usr
 BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 UNITDIR ?= $(PREFIX)/lib/systemd/system
-DSSLIBDIR ?= $(PREFIX)/lib/docker-storage-setup
+CSSLIBDIR ?= $(PREFIX)/lib/container-storage-setup
 SYSCONFDIR ?= $(DESTDIR)/etc/sysconfig
 
 .PHONY: clean
 clean:
 	-rm -rf *~ \#* .#*
 
 .PHONY: install
-install: install-docker
+install: install-docker install-core
 
 .PHONY: install-docker
-install-docker: install-core
-	install -D -m 644 ${DOCKER}-storage-setup.service ${UNITDIR}/${DOCKER}-storage-setup.service
+install-docker:
+	install -D -m 644 docker-storage-setup.service ${UNITDIR}/${DOCKER}-storage-setup.service
 	if [ ! -f ${SYSCONFDIR}/${DOCKER}-storage-setup ]; then \
-		install -D -m 644 ${DOCKER}-storage-setup-override.conf ${SYSCONFDIR}/${DOCKER}-storage-setup; \
+		install -D -m 644 docker-storage-setup-override.conf ${SYSCONFDIR}/${DOCKER}-storage-setup; \
+		echo "CONTAINER_THINPOOL=docker-pool" >> ${SYSCONFDIR}/${DOCKER}-storage-setup; \
 	fi
+	install -d -m 755 ${BINDIR}
+	(cd ${BINDIR}; ln -sf /usr/bin/container-storage-setup ${DOCKER}-storage-setup)
+	install -D -m 644 docker-storage-setup.1 ${MANDIR}/man1/${DOCKER}-storage-setup.1
 
 .PHONY: install-core
 install-core:
-	install -D -m 755 ${DOCKER}-storage-setup.sh ${BINDIR}/${DOCKER}-storage-setup
-	install -D -m 644 ${DOCKER}-storage-setup.conf ${DSSLIBDIR}/${DOCKER}-storage-setup
-	install -D -m 755 libdss.sh ${DSSLIBDIR}/libdss.sh
-	install -D -m 755 dss-child-read-write.sh ${DSSLIBDIR}/dss-child-read-write
-	install -D -m 644 ${DOCKER}-storage-setup.1 ${MANDIR}/man1/${DOCKER}-storage-setup.1
+	install -D -m 755 container-storage-setup.sh ${BINDIR}/container-storage-setup
```

Comment:
Nope since we are no longer hard coding content, docker-latest.service would use

container-storage-setup /etc/sysconfig/docker-latest-storage-setup /etc/sysconfig/docker-latest-storage